docs/form-elements/date.md

```diff
@@ -6,18 +6,21 @@
+| `hint`                   | No       | `string`  |          | A hint triggered by an icon tooltip to be displayed when hovering beside the element label.                                                                                                                                                               |
+| `defaultValue`           | No       | `string`  |          | A default value when the form is opened. Must be a string formatted as 'YYYY-MM-DD', or the string 'NOW' to denote that `defaultValueDaysOffset` will also be used to dynamically offset the `defaultValue` from the time the form is loaded.             |
+| `defaultValueDaysOffset` | No       | `number`  |          | This value must be set if `defaultValue` is set to `NOW`. It can be a negative or positive number.                                                                                                                                                        |
+| `placeholderValue`       | No       | `string`  |          | The content to appear in the form control when the form control is empty.                                                                                                                                                                                 |
+| `required`               | Yes      | `boolean` | `false`  | Determine if this input requires a date selected by the user (`true`) or not (`false`).                                                                                                                                                                   |
+| `readOnly`               | Yes      | `boolean` | `false`  | Determine if this input can be edited by the user (`false`) or not (`true`).                                                                                                                                                                              |
+| `fromDate`               | No       | `string`  |          | The earliest possible date that can be selected by the user. Must be a string formatted as 'YYYY-MM-DD', or the string 'NOW' to denote that `fromDateDaysOffset` will also be used to dynamically offset the `fromDate` from the time the form is loaded. |
+| `toDate`                 | No       | `string`  |          | The latest possible date that can be selected by the user. Must be a string formatted as 'YYYY-MM-DD', or the string 'NOW' to denote that `toDateDaysOffset` will also be used to dynamically offset the `toDate` from the time the form is loaded.       |
+| `fromDateDaysOffset`     | No       | `number`  |          | This value must be set if `fromDate` is set to `NOW`. It can be a negative or positive number, but must be less than `toDateDaysOffset` if passed.                                                                                                        |
```

Comment:
```suggestion
| `fromDateDaysOffset`     | No       | `number`  |          | This value can only be set if `fromDate` is `NOW`. It can be a negative or positive number, but must be less than `toDateDaysOffset` if passed.                                                                                                        |
```